.github/workflows/mafia_level0.yml

```diff
@@ -28,9 +28,6 @@ jobs:
       - name: BIG_CORE - rv32i_level0_no_ref -> no reference model
         run:  python build.py -dut big_core -top big_core_no_ref_tb  -test PS2_GCD -app -hw -sim -pp
       
-      - name: BIG_CORE_CACHEL1 - rv32i
```

Comment:
Dont we want to add a Sanity test for this new IP?

Reply:
yes I l know I didn't forget it, I am working now on debugging writes to cache' there is a small issue there after that I will add